app/global.scss

```diff
@@ -0,0 +1,174 @@
+:root {
+  --main-color: #242424;
+  --secondary-color: #ff3393;
+  --font-size-big: 40px;
+  --font-size-medium: 20px;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ol,
+ul {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+  border-radius: 5px;
+}
+
+@font-face {
+  src: url('../public/fonts/catamaran/catamaran bold.ttf');
+  font-family: 'UpLeveled heading font';
+  font-style: bold;
+  font-weight: 800;
+}
+
+@font-face {
+  src: url('../public/fonts/bitstream/VeraMono.ttf');
+  font-family: 'UpLeveled body';
+  font-style: normal;
+}
+
+@font-face {
+  src: url('../public/fonts/bitstream/VeraMono-Bold.ttf');
+  font-family: 'UpLeveled span';
+  font-style: bold;
+}
+
+body {
+  font-family: 'UpLeveled body', -apple-system, BlinkMacSystemFont, 'Segoe UI',
+    Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
+  font-size: var(--font-size-medium);
+  font-style: normal;
+  color: #444;
+}
+
+span {
+  font-family: 'UpLeveled span', -apple-system, BlinkMacSystemFont, 'Segoe UI',
+    Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
+  color: #1a1a1a;
+  font-style: bold;
+
+  &.counter-span {
+    margin: 0 100px;
+  }
+}
+
+h1 {
+  font-family: 'UpLeveled heading font', -apple-system, BlinkMacSystemFont,
+    'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
+  font-size: var(--font-size-big);
+  text-align: center;
+  color: var(--main-color);
+}
+
+header {
+  img {
+    padding: 5px 10px;
+    margin: 2vw 15vw;
+  }
+}
+
+img {
+  width: 15vw;
+  @media (max-width: 1100px) {
+    width: 40vw;
+  }
+}
+
+button {
+  background-color: var(--secondary-color);
+  border-radius: 25px;
+  border: none;
+  color: white;
+  cursor: pointer;
+  font-size: var(--font-size-medium);
+  max-width: 80vw;
+  padding: 0.5em;
+  width: 25em;
+
+  &:active {
+    transform: scale(1.05);
+  }
+  &:hover {
+    opacity: 0.95;
+  }
+
+  &.counter-button {
+    align-self: center;
+    width: 2em;
+  }
+}
+
+section {
+  align-items: center;
+  background-color: #d1f8cb;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  justify-content: center;
+  margin: 0 auto;
+  min-height: 20vw;
+  width: 90vw;
+}
+
+.error {
+  background-color: #f9caca;
+  color: #da0e46;
+}
+
+.productContainer {
```

Comment:
let's use CSS modules for non-global classes like this

Reply:
https://github.com/upleveled/connecting-to-external-apis-secrets-with-stripe-fall-2022-vienna-austria/blob/85f6653d67eeb9ccaf099006071fdc8d8db4124b/app/Products.tsx#L49